eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -49,20 +51,42 @@ class SqliteAuditDb(sqlite: Connection) extends AuditDb with Logging {
       statement.executeUpdate("CREATE INDEX IF NOT EXISTS channel_errors_timestamp_idx ON channel_errors(timestamp)")
     }
 
+    def migration34(statement: Statement) = {
+      statement.executeUpdate("DROP index sent_timestamp_idx")
```

Comment:
Why drop the index and recreate it later?

Reply:
Because since we're renaming the table and creating a new one, the index won't follow automatically to the new table.